clients/src/test/java/org/apache/kafka/common/requests/ProduceRequestTest.java

```diff
@@ -54,7 +55,7 @@ public void shouldBeFlaggedAsTransactionalWhenTransactionalRecords() {
         final ProduceRequest request = ProduceRequest.builder(new ProduceRequestData()
             .setTopicData(new ProduceRequestData.TopicProduceDataCollection(Collections.singletonList(
                 new ProduceRequestData.TopicProduceData()
-                    .setName("topic")
+                    .setTopicId(Uuid.fromString("H3Emm3vW7AKKO4NTRPaCWt"))
```

Comment:
Could we change the usage of `setName` in `ProducerRequestBenchmark` too?

Reply:
done